scripts/pre-down.ps1

```diff
@@ -0,0 +1 @@
+az cognitiveservices account deployment delete --name $env:AZURE_OPENAI_RESOURCE_NAME --resource-group $env:AZURE_RESOURCE_GROUP --deployment-name $env:AZURE_OPENAI_CHATGPT_DEPLOYMENT
```

Comment:
Wouldn't this need to use "azd env get-value AZURE_OPENAI_RESOURCE_NAME"? It's not necessarily in the global env- I've been trying to put as little in global env as possible lately, and fetching directly from azd when needed.

Reply:
Thanks for the suggestion. Updated.